src/Mafin.Configuration/build/Mafin.Configuration.props

```diff
@@ -0,0 +1,17 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
```

Comment:
do we really need to handle all of these file names? especially the fact that src/Mafin.Configuration/Providers/SettingsFile/SettingsFileConfigurationBuilderExtension.cs line 12 has a hardcoded file name.